service/controller/resource/prometheus/resource.go

```diff
@@ -73,6 +73,10 @@ func toPrometheus(v interface{}) (metav1.Object, error) {
 					KeyFile:  fmt.Sprintf("/etc/prometheus/secrets/%s/key", key.Secret()),
 				},
 			},
+			ExternalLabels: map[string]string{
```

Comment:
What's this for?
Also, do you not need to add reference to your `additionalScrapeConfigs` secret here?

Reply:
We have them in g8s-prometheus for https://github.com/prometheus-operator/prometheus-operator/issues/455#issue-239114745

Reply:
I am not sure what you mean about "add reference to your additionalScrapeConfigs secret here"

Reply:
Re _reference to your additionalScrapeConfigs_, never mind, I didn't see in the short diff but that's already referenced below, you know the `AdditionalScrapeConfigs: &corev1.SecretKeySelector...`.

Oh so the `ExternalLabels` are labels added to all metrics? TIL 😁